db/db-create.sql

```diff
@@ -18,12 +18,12 @@ CREATE FUNCTION change_fnc() RETURNS TRIGGER
     AS $$BEGIN
 IF TG_OP='INSERT'
 THEN
-INSERT INTO txn_history(tabname,schemaname,operation, new_val)
+INSERT INTO public.txn_history(tabname,schemaname,operation, new_val)
 VALUES(TG_RELNAME, TG_TABLE_SCHEMA, TG_OP, row_to_json(NEW));
 RETURN NEW;
 ELSIF TG_OP = 'UPDATE'
 THEN
-INSERT INTO txn_history(tabname,schemaname,operation, new_val, old_val)
+INSERT INTO public.txn_history(tabname,schemaname,operation, new_val, old_val)
 VALUES(TG_RELNAME,TG_TABLE_SCHEMA, TG_OP, row_to_json(NEW), row_to_json(OLD));
 RETURN NEW;
```

Comment:
What's the reason for specifying the schema here?

Reply:
By default, all tables are created under a "schema name" called public, it isn't necessary at all, but when I recreate the DB using backup SQL, those table name in function need this name to work. But now I removed all objects, and recreate all objects, this is not necessary now, but also it doesn't matter to leave there as now all tables are under public schema.

Reply:
If not needed we should probably delete it out then.  Or we should add it to the other table creations as well for uniformity and clarity